app/views/eforms_redirect.html

```diff
@@ -19,7 +19,7 @@ <h1 class="govuk-heading-l">Use eForms for this application</h1>
         <button class="govuk-button govuk-button--secondary" data-module="govuk-button">Apply in eForms</button>
       </form>
 
-      <a href="application-details" class="govuk-link--inverse">Back to application</a>
+      <a href="application-details" class="govuk-link--inverse">Back to your applications</a>
```

Comment:
Is the idea that the user would want to return to the current application to review/amend so that they can continue using the service?
Or that they now know that the current application is not suitable to be submitted via this service, so go back to tasklist to pick/start another?

If the former, then we shouldn't change the wording, but the href doesn't work (not sure where they should be sent)
For the latter we need to also update the href to be "tasklist"

Reply:
It's the latter. And they wouldn't go back to the task list page, they would go back to /dashboard. You're right that the href needs to be updated. I will do that now.